app/views/remove/RemoveCheckYourAnswersView.scala.html

```diff
@@ -55,7 +56,8 @@
             @p(Html(messages("remove-cya-p")))
             <div class="govuk-button-group">
                 @button("remove-cya-button")
-                @p(link(location = controllers.routes.ViewAuthorityController.onPageLoad(model.accountId, model.authorityId).url, linkMessage = "remove-cya-cancel"))
+                @p(link(location = controllers.routes.ViewAuthorityController.onPageLoad(
```

Comment:
No need to wrap the `P` component.. below should be fine.
```suggestion
               @link(location = controllers.routes.ViewAuthorityController.onPageLoad(
                    model.accountId, model.authorityId).url, linkMessage = "remove-cya-cancel")
```